otter/models/cass.py

```diff
@@ -641,7 +649,12 @@ def wrapper(*args):
             return d
         return wrapper
 
-    def view_manifest(self, with_policies=True, with_webhooks=False):
+    def _group_status(self, status, deleting):
```

Comment:
I was thinking either
a) Changing status of all `DISABLED` groups to `ERROR` and rest to `ACTIVE`
OR
b) Change code to return `ERROR` on `DISABLED` and `ACTIVE` on `None`.
What do you think?